assignment7/createClient.js

```diff
@@ -0,0 +1,24 @@
+var express = require('express');
+var router = express.Router();
+var rp = require('request-promise')
+
+router.get('/',function(req, res){
```

Comment:
@abansal26 use space after the function ().

Reply:
indentation improved. Commit ID - 2b780672efb014cb63fc8cff825b998ae65d0206